src/components/editor/index.js

```diff
@@ -4,25 +4,37 @@ import * as Action from 'common/actions';
 import FileTab from '../fileTab';
 import TextEditor from '../textEditor';
 import './styles.css';
+import { debounce } from 'lodash';
```

Comment:
You didn't installed `lodash` in the project, so it's not declared as this project's dependency. This might cause problems if the dependencies which caused the installation of lodash are removed.

Reply:
I have added the dependency to lodash. I'm not familiar with the package system of node.js. Actually I tried to update the dependencies yesterday but resulted a bunch of conflicts, which I don't know how to handle. 😊

Reply:
Yeah Github is poiting me some vulnerabilities which might be addressed with dependencies updates too... But some might have breaking changes right now...